awx/main/tests/functional/test_tasks.py

```diff
@@ -126,23 +126,6 @@ def old_version(self, control_group):
         inst.save()
         return inst
 
-    @pytest.mark.skip(reason='fix after runner merge')
-    def test_old_version(self, control_instance, old_version):
```

Comment:
We don't do version comparisons between normal AWX nodes and isolated ones because the isolated ones are just *runner* now, not some special awx version.